apps/web/next.config.js

```diff
@@ -58,7 +61,19 @@ if (process.env.GOOGLE_API_CREDENTIALS && !validJson(process.env.GOOGLE_API_CRED
   );
 }
 
+const getCachedEnglishTranslation = () => {
+  const cachedTranslation = cache.get("englishTranslation");
+  if (cachedTranslation) {
+    return cachedTranslation;
+  } else {
+    const englishTranslation = fetchEnglishTranslation();
+    cache.put("englishTranslation", englishTranslation);
+    return englishTranslation;
+  }
+};
+
 const informAboutDuplicateTranslations = () => {
+  const englishTranslation = getCachedEnglishTranslation();
```

Comment:
here englishTranslation is being stored by calling  getCachedEnglishTranslation function